serverlessworkflow/modules/ROOT/pages/integrations/custom-functions-knative.adoc

```diff
@@ -0,0 +1,335 @@
+    }
+  ]
+----
+
+In case you don't specify any path, the root path will be used. For instance, the following function:
+
+[source,json]
+----
+  "functions": [
+    {
+      "name": "greet",
+      "type": "custom",
+      "operation": "knative:custom-function-knative-service"
+    }
+  ]
+----
+
+Is exactly the same as:
+
+[source,json]
+----
+  "functions": [
+    {
+      "name": "greet",
+      "type": "custom",
+      "operation": "knative:custom-function-knative-service",
+      "metadata": {
+        "path": "/"
+      }
+    }
+  ]
+----
+
+[[referencing-custom-functions-for-knative]]
+== Referencing {context} custom functions for Knative in the Workflow DSL
+
+To reference a {context} custom functions for Knative in the Workflow DSL, you must declare a `functionRef` like the following:
+
+[source,json]
+----
+  "states": [
+    {
+      "name": "invokeFunction",
+      "type": "operation",
+      "actions": [
+        {
+          "functionRef": { <1>
+            "refName": "greet"
+          }
+        }
+      ],
+      "end": true
+    }
+  ]
+----
+<1> The `functionRef`.
+
+=== Arguments
+
+In case you need to send a payload in the request, you can add it to `arguments` in `functionRef`.
+
+For instance, given you want to send the following JSON object as the payload:
+
+[source,json]
+----
+{
+  "product_id": ".product_id",
+  "customer_name": ".customer_name"
+}
+----
+
+You must declare a `functionRef` like the following.
+
+[source,json]
+----
+  "states": [
+    {
+      "name": "invokeFunction",
+      "type": "operation",
+      "actions": [
+        {
+          "functionRef": {
+            "refName": "greet",
+            "arguments": { <1>
+              "product_id": ".product_id",
+              "customer_name": ".customer_name"
+            }
+          }
+        }
+      ],
+      "end": true
+    }
+  ]
+----
+<1> The request payload is set in `arguments`.
+
+=== Configuring the request timeout
+
+By default, the Knative service must respond within 10 seconds. You can use the `kogito.addon.knative-serving.request-timeout` property to configure this value.
+
+For instance, if you want to reduce the request timeout to 5 seconds, you must add the following to your `application.properties` file:
+
+[source,properties]
+----
+kogito.addon.knative-serving.request-timeout=5000 <1>
+----
+<1> Time in milliseconds
+
+== Example
+
+.Prerequisites
+
+include::../cloud/common/_prerequisites.adoc[]
+
+* You have the link:{kogito_sw_examples_url}/serverless-workflow-custom-function-knative/custom-function-knative-service[custom-function-knative-service] project deployed on Knative. For more information on how to deploy a Quarkus project to Knative, see the https://quarkus.io/guides/deploying-to-kubernetes[Quarkus Kubernetes extension documentation].
+
+.Procedure
+
+. Add the `knative-serving` add-on dependency to your link:{kogito_sw_examples_url}/serverless-workflow-custom-function-knative/workflow[workflow] project.
```

Comment:
The example could be mentioned earlier before the Prerequisites section, maybe. Something like we have in other docs, e.g. these steps are based on the serverless-workflow-custom-function-knative example.

Reply:
This is mentioned in the guide's first paragraph. Do you think it's necessary to mention it again here?

Reply:
@hbelmiro OK, I probably overlooked it. Thanks!